DXRModules/DeusEx/Classes/DXRMachines.uc

```diff
@@ -377,16 +383,27 @@ function RandoMedBotsRepairBots(int medbots, int repairbots)
     }
 
     medHint = 'MedbotNearby';
+    augHint = 'AugbotNearby';
     repairHint = 'RepairbotNearby';
 
     SetSeed( "RandoMedBots" );
-    if( chance_single(medbots) ) {
 #ifdef injections
-        SpawnBot(class'MedicalBot', medHint, "Medical Bot Nearby");
+        if( chance_single(medbots) ) {
+            SpawnBot(class'MedicalBot', medHint, "Medical Bot Nearby");
+        } else if ( chance_single(empty_medbots) ) {
+            mb = MedicalBot(SpawnBot(class'MedicalBot', repairHint, "Augmentation Bot Nearby"));
+            mb.augsOnly = true;
+            mb.MultiSkins[0] = Texture'AugBotTex1';
+        }
 #else
-        SpawnBot(class'DXRMedicalBot', medHint, "Medical Bot Nearby");
+        if( chance_single(medbots) ) {
+            SpawnBot(class'DXRMedicalBot', medHint, "Medical Bot Nearby");
+        } else if (( chance_single(empty_medbots)) ) {
+            mb = DXRMedicalBot(SpawnBot(class'DXRMedicalBot', repairHint, "Augmentation Bot Nearby"));
+            mb.augsOnly = true;
+            mb.MultiSkins[0] = Texture'AugBotTex1';
```

Comment:
You could probably make these two changes to the medbot a convenience function inside DXRMedicalBot (something like "MakeAugOnly()" that sets augsOnly and the skin)